go/apps/api/routes/v2_keys_reroll_key/handler.go

```
@@ -0,0 +1,374 @@
+		// Calculate the desired expiry time (rounded up to next minute)
+		expiration := time.Now().Add(time.Millisecond * time.Duration(req.Expiration))
+		// Round up to next minute (ceil)
+		if expiration.Truncate(time.Minute) != expiration {
+			expiration = expiration.Truncate(time.Minute).Add(time.Minute)
+		}
+
+		if req.Expiration == 0 {
+			expiration = time.Now()
+		}
+
+		err = db.Query.UpdateKey(ctx, tx, db.UpdateKeyParams{
```

golangci-lint (2.2.2)

db.UpdateKeyParams is missing fields NameSpecified, Name, IdentityIDSpecified, IdentityID, EnabledSpecified, Enabled, MetaSpecified, Meta, RemainingRequestsSpecified, RemainingRequests, RefillAmountSpecified, RefillAmount, RefillDaySpecified, RefillDay, Now

(exhaustruct)